### PR TITLE
Explain how to invalidate API tokens

### PR DESCRIPTION
**Pull request by catherineluse**
This PR explains the way to invalidate API tokens as described in this issue: https://github.com/rancherlabs/rancher-security/issues/239

### CODE REVIEW

File: content/rancher/v2.x/en/api/_index.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ The API has its own user interface accessible from a web browser.  This is an ea
 
 API requests must include authentication information.  Authentication is done with HTTP basic authentication using [API Keys]({{< baseurl >}}/rancher/v2.x/en/user-settings/api-keys/). API keys can create new clusters and have access to multiple clusters via `/v3/clusters/`. [Cluster and project roles]({{< baseurl >}}/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/) apply to these keys and restrict what clusters and projects the account can see and what actions they can take.
 
+By default, all cluster-level API tokens are generated with infinite time-to-live (`ttl=0`). In other words, API tokens never expire unless you invalidate them. For details on how to invalidate them, refer to the [API tokens page]({{<baseurl>}}/rancher/v2.x/en/api/api-tokens).
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/api/_index.md]: i don't believe it's *all*, but some cluster API tokens.

File: content/rancher/v2.x/en/api/api-tokens/_index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: API Tokens
+weight: 1
+---
+
+By default, all cluster-level API tokens are generated with infinite time-to-live (`ttl=0`). In other words, API tokens never expire unless you invalidate them. Tokens are not invalidated by changing a password.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/api/api-tokens/_index.md]: remove "all"

 In other words, some API tokens never expire...

Put the "Tokens are note validated sentence in a note"

File: content/rancher/v2.x/en/api/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/api/_index.md]: fixed

File: content/rancher/v2.x/en/api/api-tokens/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/api/api-tokens/_index.md]: fixed

File: content/rancher/v2.x/en/api/api-tokens/_index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: API Tokens
+weight: 1
+---
+
+By default, some cluster-level API tokens are generated with infinite time-to-live (`ttl=0`). In other words, API tokens with `ttl=0` never expire unless you invalidate them. Tokens are not invalidated by changing a password.
+
+You can deactivate API tokens by deleting them or by deactivating the user account.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/api/api-tokens/_index.md]: @catherineluse can you confirm with @cjellick that deactivating the user account would invalidate the token?

Comment (catherineluse) [reply content/rancher/v2.x/en/api/api-tokens/_index.md]: Craig confirmed that it invalidates the tokens

**Pull request merged**